agora_analytics/manifest.json

```diff
@@ -0,0 +1,59 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "a752523e-9c3d-458c-a91a-af0c9fae5adc",
+  "app_id": "agora-analytics",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "View Agora Analytics Collector metrics in Datadog",
+    "title": "Agora Analytics",
+    "media": [
+      {"media_type": "image",
+        "caption": "Agora Analytics Overview - Dashboard",
+        "image_url": "images/agora_analytics_dashboard.png"
```

Comment:
Thanks for adding this dashboard! Feel free to remove the picture of the individual graph. Could you also add a description to the dashboard so customers know what to expect? You can do so by clicking on the title while editing the dashboard.
<img width="594" alt="Screen Shot 2023-02-17 at 9 48 58 AM" src="https://user-images.githubusercontent.com/109224841/219688514-d1267fe0-d1c9-4c15-b078-7cb88e9208b8.png">

Reply:
Thanks for suggestions. I added description to the dashboard and removed the individual graph.